src/loaders/express.js

```diff
@@ -23,33 +25,35 @@ module.exports = (app) => {
         },
     }));
 
-    // Applying middleware
     // JSON bodyparser (parses JSON request body into req.body)
     app.use(bodyParser.json());
 
-
+    // Initializing passport authentication settings
     app.use(passport.initialize());
     app.use(passport.session());
 
-    // Adding headers
-    app.use((req, res, next) => {
-    // Website to allow to connect
-        res.setHeader("Access-Control-Allow-Origin", "*");
+    // Adds route logging
+    app.use(morgan("common"));
 
-        // Request methods to allow
+    // Adding headers (CORS)
+    app.use((_, res, next) => {
+        // Allow connections for all origins
+        res.setHeader("Access-Control-Allow-Origin", "*");
```

Comment:
We should have an `env` variable with the allowed origins, for more safety. It is not that hard and we could customize it to allow only our applications for now (nijobs-fe, for example). Surely for development environment, we could have the `*` the same way.

Reply:
This is a valid future concern, please create an issue mentioning your comment for increased context.

Reply:
Here you go: #41

Reply:
Thanks!